.changeset/gentle-kiwis-admire.md

```diff
@@ -0,0 +1,5 @@
+---
+"apollo-client-devtools": minor
+---
+
+Add functionality to inject error and log messages.
```

Comment:
```suggestion
Add the ability for devtools to provide full error and log messages without the need to need to call `loadDevMessages` or `loadErrorMessages`.
```

What about making this a bit more verbose to really make this feature stand out? For those reading the changelog, I think this might pique interest a bit more. Thoughts?